packages/functional-tests/pages/relier.ts

```diff
@@ -8,8 +8,10 @@ export class RelierPage extends BaseLayout {
     return this.page.goto(url);
   }
 
-  isLoggedIn() {
-    return this.page.isVisible('#loggedin', { timeout: 1000 });
+  async isLoggedIn() {
+    const login = this.page.locator('#loggedin');
+    await login.waitFor();
```

Comment:
I changed this function a little, because it was being flaky with the `.isVisible()`. The `locator()` I added doesn't seem to allow an explicit timeout to be set specifically for this one, I will have to alter all the default timeouts used everywhere (or that's what I found). Hence I had to go with the default 30s.